pyproject.toml

```diff
@@ -1,11 +1,84 @@
 [build-system]
-# Setuptools version constraint ensures PEP 517 compatibility
-requires = [
-    "setuptools >= 45",
-    "wheel",
-]
+requires = [ "setuptools>=61.2" ]
 build-backend = "setuptools.build_meta"
 
+[project]
```

Comment:
Awesome that we are switching to this!

I strongly suggest adding validate-pyproject to the linting process, like the other projects do, to help ensure the integrity/consistency of this file. Also check that the order of configuration blocks matches our other projects (validate-pyproject does not check for any particular order).

Reply:
Good idea, added in 77b9e62